ballerina/Ballerina.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+org = "ballerinax"
+name = "sap.jco"
+version = "0.1.0"
+distribution = "2201.9.0"
+authors = ["Ballerina"]
+keywords = ["Business Management/ERP", "Cost/Paid", "Vendor/SAP"]
+repository = "https://github.com/ballerina-platform/module-ballerinax-sap.jco"
+icon = "icon.png"
+license = ["Apache-2.0"]
+
+[build-options]
+observabilityIncluded = true
+
+[platform.java17]
+graalvmCompatible = true
```

Comment:
Were these properties tested? For now lets remove them and commit

Reply:
Changed with [9983b0b](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/1/commits/9983b0b7a54c408c1721d199a19a3164283937f4).